backend/question/docs/question.yaml

```diff
@@ -0,0 +1,300 @@
+        "200":
+          description: List of questions
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QuestionAll"
+        "500":
+          description: Internal server error
+
+  /{id}:
+    get:
+      summary: Get a specific question by ID
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+          description: ID of the question
+      responses:
+        "200":
+          description: A specific question
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "400":
+          description: Invalid ID format
+        "404":
+          description: Question not found
+        "500":
+          description: Internal server error
+
+  /{id}/update:
+    post:
+      summary: Update a specific question by ID
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+          description: ID of the question to update
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/QuestionUpdate"
+      responses:
+        "200":
+          description: Updated question
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QuestionUpdate"
+        "400":
+          description: Validation or duplicate title error
+        "404":
+          description: Question not found
+        "500":
+          description: Internal server error
+
+  /{id}/delete:
+    post:
+      summary: Delete a specific question by ID
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+          description: ID of the question to delete
+      responses:
+        "200":
+          description: Deleted question
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "404":
+          description: Question not found
+        "500":
+          description: Internal server error
+
+components:
+  schemas:
+    Question:
+      type: object
+      properties:
+        questionid:
+          type: integer
+          example: 1
+        title:
+          type: string
+          example: "Two Sum"
+        description:
+          type: string
+          example: "Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.You may assume that each input would have exactly one solution, and you may not use the same element twice.You can return the answer in any order."
+        category:
+          type: array
+          items:
+            type: string
+          example: ["Array, Hash Table"]
```

Comment:
Typo in the example here? Should be ["Array", "Hash Table"], while your example is technically valid. Same as Line 200

Reply:
You are right, the example has been fixed!